example/test_driver/app_test.dart

```diff
@@ -0,0 +1,52 @@
+// Imports the Flutter Driver API.
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Webcrypto Example App', () {
+    final keyOutputFinder = find.byValueKey('KeyOutput');
+    final importKeyButtonFinder = find.byValueKey('ImportRawKey');
+    final genKeyOutputFinder = find.byValueKey('GenKeyOutput');
+    final generateKeyButtonFinder = find.byValueKey('GenerateKey');
+
+    FlutterDriver driver;
+
+    // Connect to the Flutter driver before running any tests.
+    setUpAll(() async {
+      driver = await FlutterDriver.connect();
+    });
+
+    // Close the connection to the driver after the tests have completed.
+    tearDownAll(() async {
+      if (driver != null) {
+        driver.close();
+      }
+    });
+
+    test('no imported key', () async {
+      // Verify that no key has been imported in the begining.
+      expect(await driver.getText(keyOutputFinder), "-");
+    });
+
+    test('import raw aes key', () async {
+      // Tap the button to import the key.
+      await driver.tap(importKeyButtonFinder);
+
+      // The outout should contain the importet key now.
+      expect(await driver.getText(keyOutputFinder), '{"kty":"oct","alg":"A128GCM","k":"3nle6RpFx77jwrksoNUb1Q"}');
```

Comment:
This fails for me, maybe just:
```suggestion
      expect(await driver.getText(keyOutputFinder),
          contains('3nle6RpFx77jwrksoNUb1Q'));
```

Some browsers might include other values and change ordering of JSON.
We don't need an accurate test here, just hint that it isn't totally wrong :D

Reply:
You are totally right, there is no need for an accurate test in this case. The test has been adjusted.